### DISCUSSION TITLE
Android/iOS Phone/Pad support? Gyro aiming?

### DISCUSSION THREAD

**Discussion by complyue** [AmbientRun/Ambient#224] [Ideas]
I have multiplayer p2p/LAN FPS with mobile devices in mind, also enjoying gyro experience like that in CoD mobile, how likely you'll consider to support such apps?

**Comment by FredrikNoren** [comment]
@complyue We would love to add support for this, but we're not actively working on it right now so I can't say exactly when it will happen. But it's on our wish list.
